core/src/styles/utils/_toast.scss

```diff
@@ -0,0 +1,72 @@
+@import 'variables';
+@import 'mixins';
+
+.toastify {
```

Comment:
do you need this to increase selector specificity? otherwise just go with .cat-toastify